app/views/teachers/teacher-dashboard/BaseResourceHub/index.vue

```diff
@@ -0,0 +1,277 @@
+      }),
+      trackEvent (eventName) {
+        if (eventName) {
+          window.tracker?.trackEvent(eventName, { category: 'Teachers' })
+        }
+      },
+    }
+  }
+</script>
+
+<template>
+  <div id="base-resource-hub">
+    <div class="flex-container">
+      <div class="aside">
+        <h4>{{ $t('common.table_of_contents') }}</h4>
+        <ul>
+          <li v-for="resourceHubSection in resourceHubSections">
+            <a v-if="resourceHubLinks(resourceHubSection.sectionName).length" :href="'#' + resourceHubSection.slug">{{ $t(resourceHubSection.i18nKey) }}</a>
+          </li>
+        </ul>
+
+        <h4>{{ $t('nav.contact') }}</h4>
+        <div class="contact-icon">
+          <img src="/images/ozaria/teachers/dashboard/svg_icons/IconMail.svg">
+          <a
+            :href="`mailto:${$t('teacher_dashboard.support_coco')}`"
+            @click="trackEvent('Resource Hub: Support Email Clicked')"
+          >
+            {{ $t('teacher_dashboard.support_coco') }}
+          </a>
+        </div>
+      </div>
+
+      <div class="resource-hub">
+        <div class="resource-hub-section" v-for="resourceHubSection in resourceHubSections" :id="resourceHubSection.slug">
+        <h4 v-if="resourceHubLinks(resourceHubSection.sectionName).length">
+          {{ $t(resourceHubSection.i18nKey) }}
+        </h4>
+          <div class="resource-contents-row">
+            <button-resource-icon
+              v-for="resourceHubLink in resourceHubLinks(resourceHubSection.sectionName)"
+              :key="resourceHubLink.name"
+              :icon="resourceHubLink.icon"
+              :label="resourceHubLink.name"
+              :link="resourceHubLink.link"
+              :description="resourceHubLink.description"
+              :locked="resourceHubLink.locked"
+              :from="resourceHubLink.source || 'Resource Hub'"
+              :section="resourceHubSection.slug"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+#base-resource-hub {
+  margin-bottom: -50px;
+}
+
+.contact-icon {
+  display: flex;
+  flex-direction: row;
+
+  img {
+    margin-right: 10px;
+  }
+
+  a {
+    font-size: 14px;
+    line-height: 18px;
+    letter-spacing: 0.2667px;
+    text-decoration: none;
+  }
+}
+
+.flex-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.aside {
+  margin-top: -20px;
+
+  width: 285px;
+  padding: 30px;
+
+  background-color: #f2f2f2;
+  box-shadow: -1px 0px 1px rgba(0, 0, 0, 0.06), 3px 0px 8px rgba(0, 0, 0, 0.15);
+
+  h4 {
+    color: #131b25;
+    font-family: 'Work Sans';
+    font-size: 18px;
+    line-height: 30px;
+    letter-spacing: 0.44px;
+    font-weight: 600;
+    text-transform: uppercase;
+    margin-bottom: 15px;
+  }
+
+  ul a {
+    text-decoration: underline;
+  }
+}
+
+.aside ul {
+  padding: 0;
+  list-style: none;
+
+  margin-bottom: 50px;
+}
+
+.resource-hub {
+  padding: 40px 30px;
```

Comment:
Not sure if this is where it's coming in on a quick look, but the screenshot shows too much top margin/padding on the first section; the first "Getting Started" section header on the right should be top-aligned with "TABLE OF CONTENTS" section header on the left.